src/dcap_provider.cpp

```diff
@@ -135,6 +138,25 @@ static std::string get_collateral_version()
     }
 }
 
+static std::string get_agent_base_url()

```

Comment:
Is this function necessary? It made sense with THIM cloud service because we could test different cloud endpoints (e.g. Europe vs. Americas). But THIM Agent will always be hosted at the same local IP address.

Reply:
I think this will be useful for dev box testing with IMDS and THIM agent running on the dev box.